deployment/ansible/group_vars/all

```diff
@@ -3,7 +3,9 @@ app_username: "vagrant"
 
 packer_version: "0.7.5"
 
-nodejs_version: 4.6.0
+nodejs_version: 4.6.2
+
+virtualenv_version: 15.1.0
```

Comment:
Oh, now I remember. Ran into: azavea/ansible-virtualenv#1

So not totally certain if this version bump was necessary, but I think it did move me past that to a different error.